File: NapakalakiRuby/lib/treasure.rb
module NapakalakiGame
class Treasure
	attr_accessor :name, :bonus, :enum
	def initialize(name, bonus, enum)
		@name = name
		@bonus = bonus
		@enum = enum
	end

	def getName
		@name
	end

	def getBonus
		@bonus
	end

	def getType
		@enum
	end

end
end

File: Napakalaki/src/NapakalakiGame/Treasure.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open tNe template in the editor.
 */
package NapakalakiGame;
import java.util.ArrayList;


public class Treasure{
	private String name;
	private int bonus;
	private TreasureKind enume;

	Treasure (String n, int bonus, TreasureKind t){
		name = n;
		this.bonus = bonus;
		enume = t;
	}

	public String getName(){
		return name;
	}

	public int getBonus(){
		return bonus;
	}

	public TreasureKind getType(){
		return enume;
	}
        
        public String toString(){
        return "Nombre = " +name+" bonus="+ Integer.toString(bonus);
    }

}